web-ui/src/main/resources/templates/starter.html

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```

Comment:
doesn't seem to be related to https://github.com/Tutritp01/TgBot_WebClient/pull/16, does it?